src/EntityManager.cpp

```diff
@@ -590,12 +590,18 @@ void postToDbus(const nlohmann::json& newConfiguration,
             boardType = findBoardType->get<std::string>();
             std::regex_replace(boardType.begin(), boardType.begin(),
                                boardType.end(), illegalDbusMemberRegex, "_");
+            if (boardType == "Chassis")
+            {
+                std::cout << "Ignoring chassis type" << std::endl;
+                continue;
```

Comment:
Wouldn't doing this continue here cause us to skip all of the other properties for the part? The goal here is to only skip hosting the xyz.openbmc_project.Inventory.Item.Chassis interface, any other configuration must still be exposed on D-Bus.